sonar-db/src/test/java/org/sonar/db/event/EventDaoTest.java

```diff
@@ -60,15 +64,16 @@ public void select_by_component_uuid() {
   public void return_different_categories() {
     dbTester.prepareDbUnit(getClass(), "shared.xml");
 
-    List<EventDto> dtos = dao.selectByComponentUuid(dbTester.getSession(), "ABCD");
+    List<EventDto> dtos = underTest.selectByComponentUuid(dbTester.getSession(), "ABCD");
     assertThat(dtos).extracting("category").containsOnly(EventDto.CATEGORY_ALERT, EventDto.CATEGORY_PROFILE, EventDto.CATEGORY_VERSION);
   }
 
   @Test
   public void insert() {
     dbTester.prepareDbUnit(getClass(), "empty.xml");
```

Comment:
Nice to have, it would have been good to remove DbUnit, as the goal is to stop using it (too much a pain to maintain).
Note that for migration it can still makes sense, as migration never change.